docs/es/latest/FAQ.md

```diff
@@ -0,0 +1,357 @@
+
+[Cómo escribir un plug-in](doc/plugin-develop.md)
+
+## ¿Por qué elegimos etcd como el centro de la configuración?
+
+Para el centro de la configuración, la configuración del almacenamiento es solamente la función más básica, y APISIX necesita también las siguientes prestaciones:
+
+1. Grupos (Cluster)
+2. Transacciones
+3. Control de concurrencia multi-versión
+4. Notificación de cambios
+5. Alto rendimiento
+
+Más información en [Por qué etcd](https://github.com/etcd-io/etcd/blob/master/Documentation/learning/why.md#comparison-chart).
+
+## ¿Por qué sucede que instalar dependencias APISIX con Luarocks provoca interrupciones por exceso de tiempo (timeout), o instalaciones lentas y fallidas?
+
+Existen dos posibilidades cuando encontramos Luarocks muy lentos:
+
+1. El servidor usado para instalar Luarocks está bloqueado
+2. En algún punto entre su red y el servidor de github se bloquea el protocolo 'git'
+
+Para el primer problema usted puede usar https_proxy o usar la opción `--server` para especificar un servidor de Luarocks al que usted pueda acceder con mayor velocidad.
+Ejecute el comando `luarocks config rocks_servers` (este comando es soportado por versiones posteriores a luarocks 3.0) para ver qué servidores están disponibles.
+
+Si usar un proxy no resuelve este problema, usted puede agregar la opción `--verbose` durante la instalación para ver qué tan lento está. Excluyendo el primer caso, solamente en el segundo, cuando el protocolo `git` está bloqueado, podemos ejecutar `git config --global url."https://".insteadOf git://` para usar el protocolo 'HTTPS' en lugar de `git`.
+
+## ¿Cómo se soporta un lanzamiento en etapa "gray release" (lanzamiento gris) a través de APISIX?
+
+Un ejemplo, `foo.com/product/index.html?id=204&page=2`, lanzamiento gris (gray release) basado en `id` en la cadena de consulta (query string) en URL como una condición：
+
+1. Grupo A：id <= 1000
+2. Grupo B：id > 1000
+
+Hay dos formas diferentes de hacer esto：
+
+1. Usar el campo `vars` de la ruta para hacerlo.
+
+```shell
+curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "uri": "/index.html",
+    "vars": [
+        ["arg_id", "<=", "1000"]
+    ],
+    "plugins": {
+        "redirect": {
+            "uri": "/test?group_id=1"
+        }
+    }
+}'
+
+curl -i http://127.0.0.1:9080/apisix/admin/routes/2 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "uri": "/index.html",
+    "vars": [
+        ["arg_id", ">", "1000"]
+    ],
+    "plugins": {
+        "redirect": {
+            "uri": "/test?group_id=2"
+        }
+    }
+}'
+```
+
+Aquí encontramos la lista de operadores del `lua-resty-radixtree` actual：
+https://github.com/iresty/lua-resty-radixtree#operator-list
+
+2. Usar el plug-in `traffic-split` para hacerlo.
+
+Por favor consultar la documentación de plug-in [traffic-split.md](doc/plugins/traffic-split.md) para ver ejemplos de uso.
```

Comment:
Hi, I noticed that there have some outdated paths, we could use something like `https://apisix.apache.org/docs/apisix/plugins/xxx` instead if it doesn't exit IMO.